materials/spraymanager/transparent.vmt

```diff
@@ -1,8 +1,8 @@
-"LightmappedGeneric"
-{
-	"$basetexture"	"decals\spraymanager\3"
-	"$translucent" 	"1"
-    	"$decal" 		"1"
-	"$decalscale" 	1.000000
-    	"$surfaceprop" 	"Default"
+"LightmappedGeneric"
+{
+	"$basetexture"	"decals\spraymanager\3"
```

Comment:
i think this needs to be changed no?

Reply:
Yes, no clue why it did not take in it in count.